efel/tests/test_basic.py

```diff
@@ -642,8 +642,10 @@ def test_min_AHP_indices_single_peak():
         [trace], ["min_AHP_values", "min_AHP_indices", "peak_indices"])
 
     assert len(feats[0]["peak_indices"]) == 1
-    assert feats[0]["min_AHP_indices"] is None
-    assert feats[0]["min_AHP_values"] is None
+    assert len(feats[0]["min_AHP_indices"]) == 1
+    assert len(feats[0]["min_AHP_values"]) == 1
+    assert feats[0]["min_AHP_indices"][0] == 499
+    assert abs(feats[0]["min_AHP_values"][0] - (-77.5954895)) < 1e-5
```

Comment:
now the single peak data is returning a minima.

Reply:
It looks to me like it was not returning a minima before, because the 'real' minima was outside the interval (after the end of the stimulus). If we return a minima that is above what would have been returned if the simulation lasted longer / the interval were larger, I am afraid it might mislead the scientists, no?

Reply:
I think @AurelienJaquier is correct.